### ISSUE TITLE
Get wrong result after add BatchNormalization layer

### ISSUE THREAD

**Issue by RexHuang** [tensorflow/tfjs#1259]
Get wrong result after add BatchNormalization layer

#### TensorFlow.js version
0.14.1

#### Browser version
Google Chrome 72.0.3626.109

#### Describe the problem or feature request
I am using Tensorflow.js to predict on a model I trained in Keras, use the following model is ok:

model = Sequential()
model.add(Conv2D(filters = 32, kernel_size = (5,5),padding = 'Same', activation ='relu', input_shape = (28,28,1)))
model.add(Conv2D(filters = 32, kernel_size = (5,5),padding = 'Same', activation ='relu'))
model.add(MaxPool2D(pool_size=(2,2)))
model.add(Dropout(0.25))
model.add(Conv2D(filters = 64, kernel_size = (3,3),padding = 'Same', activation ='relu'))
model.add(Conv2D(filters = 64, kernel_size = (3,3),padding = 'Same', activation ='relu'))
model.add(MaxPool2D(pool_size=(2,2), strides=(2,2)))
model.add(Dropout(0.25))
model.add(Flatten())
model.add(Dense(256, activation = "relu"))
model.add(Dropout(0.5))
model.add(Dense(10, activation = "softmax")) 

But it will get wrong result after I add BatchNormalization layer, this model is work in python:

model = Sequential()
model.add(Conv2D(filters = 32, kernel_size = (5,5),padding = 'Same', activation ='relu', input_shape = (28,28,1)))
model.add(BatchNormalization())
model.add(Conv2D(filters = 32, kernel_size = (5,5),padding = 'Same', activation ='relu'))
model.add(BatchNormalization())
model.add(MaxPool2D(pool_size=(2,2)))
model.add(Dropout(0.25))
model.add(Conv2D(filters = 64, kernel_size = (3,3),padding = 'Same', activation ='relu'))
model.add(BatchNormalization())
model.add(Conv2D(filters = 64, kernel_size = (3,3),padding = 'Same', activation ='relu'))
model.add(BatchNormalization())
model.add(MaxPool2D(pool_size=(2,2), strides=(2,2)))
model.add(Dropout(0.25))
model.add(Flatten())
model.add(Dense(256, activation = "relu"))
model.add(Dropout(0.5))
model.add(Dense(10, activation = "softmax")) 

#### Code to reproduce the bug / link to feature request
I have put the code in github to reproduce it:
https://github.com/RexHuang/tfjs_issues_BatchNormalization

**Comment by caisq** [comment]
@RexHuang One thing you might have noticed is that the implementation of BatchNormalization differs between keras and tensorflow.keras. Our implementation in TF.js aligns with tensorflow.keras. So my guess is that you are using keras (i.e., not tensorflow.keras) and that's the reason for the discrepancy you see. Can you confirm what "flavor" of keras you are using?

**Comment by RexHuang** [comment]
Here's the BatchNormalization I am using:
from keras.layers.normalization import BatchNormalization

**Comment by RexHuang** [comment]
@caisq What's your suggestion about the fix?

**Comment by caisq** [comment]
@RexHuang For now, please use the keras that comes with tensorflow, instead of keras-team/keras. In other words, use

`from tensorflow.keras.layers.normalization import BatchNormalization`

All other important may need to be changed accordingly.

**Comment by RexHuang** [comment]
@caisq it works after change to tensorflow.keras, thank you very much !!
